examples/todo.rs

```diff
@@ -72,31 +79,39 @@ fn main() {
         ..Server::default()
     }.run();
 
-    if let Err(e) = server_result {
-        error!("could not run the server: {}", e)
-    }
+    match server_result {
+      Ok(server) => {
+        println!(
+          "This example is a showcase implementation of Todo-Backend project (http://todobackend.com/), \
+          visit http://localhost:{}/ to try it or run reference test suite by pointing \
+          your browser to http://todobackend.com/specs/index.html?http://localhost:{}",
+          server.socket.port(), server.socket.port()
```

Comment:
There's a neat trick that can be used here: replace `{}` with `{0}` to take the first parameter in both of the placeholders. No need to repeat it.

See https://doc.rust-lang.org/std/fmt/ for more. :smile: